src/QsCompiler/CommandLineTool/Commands/Build.cs

```diff
@@ -92,11 +92,12 @@ public static int Run(BuildOptions options, ConsoleLogger logger)
             var specifiesTargets = options.Targets != null && options.Targets.Any();
             var loadOptions = new CompilationLoader.Configuration
             {
-                ProjectFile = options.ProjectName == null ? null : new Uri(Path.GetFullPath(options.ProjectName)),
+                ProjectName = options.ProjectName,
                 GenerateFunctorSupport = true,
                 SkipSyntaxTreeTrimming = false,
                 DocumentationOutputFolder = options.DocFolder,
                 BuildOutputFolder = options.OutputFolder ?? (specifiesTargets ? "." : null),
+                DllOutputPath = " ", // generating the dll in the same location as the .bson file
```

Comment:
this is a blank space (vs an empty string). is that on purpose?

Reply:
Both works; a dll will be generated as long as the DllOutputPath is not null, and a default path will be chosen if it is empty or whitespace.